catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/ResourceOperationsTest.java

```diff
@@ -374,8 +361,6 @@ public void testGetResourceInfoWithFragment() throws Exception {
     ResourceOperations resourceOperationsSpy = spy(resourceOperations);
     resourceOperationsSpy.getResourceInfo(
         resourceRequestMock, "site", isEnterprise, federatedSite, requestProperties, false);
-    verify(resourceOperationsSpy)
-        .createPropertyStartsWithQuery(Metacard.RESOURCE_URI, "bobUri:test");
   }
```

Comment:
I just realized this doesn't seem to be testing anything . Can I change this to do something meaningful or should I just delete it?

Reply:
can you just change this to verify createPropertyEqualToQuery?

Reply:
Same for the two cases below as well